src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1093,7 +1093,6 @@ export class ResultsViewPageStore extends AnalysisStore
             this.studyIds,
             this.clinicalAttributes_profiledIn,
             this.clinicalAttributes_comparisonGroupMembership,
-            this.clinicalAttributes_customCharts,
```

Comment:
be careful of removing `this.clinicalAttributes_customCharts` here. this may cause some unintended effects when `clinicalAttributes` is used. 
for example, the tracks menu in the Oncoprint tab of results view is not showing the custom attributes anymore. let's fix these issues or add back `this.clinicalAttributes_customCharts` and find a different approach.

Reply:
Added back